pytest-selenium/pages/content.py

```diff
@@ -181,6 +232,10 @@ def header(self):
         def toc(self):
             return TableOfContents(self.page)
 
+        @property
+        def search_sidebar(self):
```

Comment:
this seems like a weird place to put this... everything else in the SideBar class is all about the toc, and the root locator for the SideBar class is for the toc

Reply:
Agreed and I forgot that sidebar (TOC) and search sidebar are different.
I have now added it directly in the content and did not create a separate class for search_sidebar. So in tests its now called as search_sidebar = content.search_sidebar
Let me know if this is fine or let me know what other place would best fit it